.rubocop.yml

```diff
@@ -0,0 +1,15 @@
+AllCops:
+  TargetRubyVersion: 2.4
+  Include:
+    - '**/Rakefile'
+    - '**/config.ru'
+  Exclude:
+    - db/schema.rb
+    - 'config/**/*'
+    - 'script/**/*'
+Documentation:
+  Enabled: false
+Rails:
+  Enabled: true
+Metrics/LineLength:
+  Max: 140
```

Comment:
I think this is one that could have most opinions against... but default value its `80` too short for me, nowadays most of screens are wider. But no hard opinions about it... just "convinient"